packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -609,6 +663,56 @@ describe('portfolio associations and product constraints', () => {
       }).toThrowError(/Cannot set multiple launch roles for association/);
     }),
 
+    test('local launch role must have roleName explicitly set', () => {
+      const otherLaunchRole = new iam.Role(stack, 'otherLaunchRole', {
+        assumedBy: new iam.ServicePrincipal('servicecatalog.amazonaws.com'),
+      });
+
+      expect(() => {
+        portfolio.setLocalLaunchRole(product, otherLaunchRole);
+      }).toThrowError(/Role otherLaunchRole used for Local Launch Role must have roleName explicitly set/);
```

Comment:
`toThrow()` is fine, we don't need to assert it's an `Error`.

Same comment in the tests below.